diagnostic.md

```diff
@@ -76,7 +76,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
+_A stick of butter_
```

Comment:
Remember that case matters in computer programming.  "a sticker of butter" is not the same as "A stick of butter" to the computer.